src/client/components/HyfBigLogo/HyfBigLogo.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './HyfBigLogo.style.css';
+import hyfLogo from './hyfLogo.png';
+import { PropTypes } from 'prop-types';
+
+export default function HyfBigLogo({ title }) {
+  return (
+    <div className="hyf">
+      <div>
+        <img className="hyf-logo" src={hyfLogo} alt="logo" />
+      </div>
+      <h1 className="app-title">{title}</h1>
+    </div>
+  );
+}
+
+HyfBigLogo.propTypes = {
+  title: PropTypes.string.isRequired,
+};
```

Comment:
I think default prop types is missing here

Reply:
@dpfernandes this was supposed for default prop types ....but can understand this doesn't make any sense(after Toni's comments).